nxpprog.py

```diff
@@ -464,27 +464,10 @@ def __init__(self, device, baud, xonxoff=False, control=False):
     # this is of course only possible if the signals are connected in
     # this way
     def isp_mode(self):
-        self.reset(0)
-        time.sleep(.1)
-        self.reset(1)
-        self.int0(1)
-        time.sleep(.1)
-        self.reset(0)
-        time.sleep(.1)
-        self.int0(0)
-
-    def reset(self, level):
-        if self.reset_pin == "rts":
-            self._serial.setRTS(level)
-        else:
-            self._serial.setDTR(level)
-
-    def int0(self, level):
-        # if reset pin is rts int0 pin is dtr
-        if self.reset_pin == "rts":
-            self._serial.setDTR(level)
-        else:
-            self._serial.setRTS(level)
+        self._serial.setRTS(1) # set RTS line to 0v
```

Comment:
Please add a comment here referring to the AFCv4 board: [https://ohwr.org/project/afc/tree/cti_afc4](https://ohwr.org/project/afc/tree/cti_afc4)